src/pages/Sources.js

```diff
@@ -264,6 +269,16 @@ const SourcesPage = () => {
           filters: prepareChips(filterValue, sourceTypes, appTypes, intl),
           onDelete: (_event, chips, deleteAll) => dispatch(filterSources(removeChips(chips, filterValue, deleteAll))),
         }}
+        exportConfig={{
+          isDisabled: !loaded,
+          onSelect: (_e, type) => {
+            const data =
+              type === 'csv'
+                ? generateCSV(entities, intl, appTypes, sourceTypes)
+                : generateJSON(entities, intl, appTypes, sourceTypes);
+            downloadFile(data, `sources-${new Date().toISOString()}`, type);
```

Comment:
As this is very dependent on the API response shape and can cause runtime errors, can we add a try-catch block around the action? And possibly display error notifications if the CSV/JSON conversion fails?

Reply:
If the response is different than the whole app would fail. 🤷

Reply:
Spicy :hot_pepper:

Reply:
There is the error boundary, so it should be ok. 😏 

But the whole application is based on this one API response, so if it were changed, then the whole application would be changed and there are integration tests that would catch all the differences.